homeassistant/util/json.py

```diff
@@ -8,10 +8,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-_UNDEFINED = object()
+_UNDEFINED = -1
 
 
-def load_json(filename: str, default: Union[List, Dict] = _UNDEFINED) \
+def load_json(filename: str, default: Union[List, Dict, int] = _UNDEFINED) \
```

Comment:
Why is it better to use `_UNDEFINED` than `None`?

Reply:
Changed to None